cat/CaT-service.yaml

```diff
@@ -3296,7 +3296,34 @@ components:
               type: string
               format: date-time
               description: Date/time - Event Created
-
+            buyerQuestions:
+              type: array
+              items:
+                allOf:
+                  - $ref: '#/components/schemas/EvalCriteria'
+                  - type: object
```

Comment:
Hi @ronin1972 - I just had a run through the generator to have a look at the java classes and this approach may not work (i.e. combining a $ref and object in this manner). The generator will just see this as an array of type `EvalCriteria` (even though it looks ok in the swagger editor).

I think we may either need to extend `EvalCriteria` to include a list of `QuestionGroups` (and likewise add a list of `Questions` to each `QuestionGroup`), or create equivalent wrapper objects to reference here.

Let me just check with @tommyb82 to see if he's aware of any workaround.